gadgets/fsnotify/test/unit/fsnotify_test.go

```diff
@@ -0,0 +1,230 @@
+// Copyright 2024 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tests
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"syscall"
+	"testing"
+	"time"
+
+	"github.com/fsnotify/fsnotify"
+
+	gadgettesting "github.com/inspektor-gadget/inspektor-gadget/gadgets/testing"
+	utilstest "github.com/inspektor-gadget/inspektor-gadget/internal/test"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/operators"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/testing/gadgetrunner"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/testing/utils"
+)
+
+const TASK_COMM_LEN = 16
+
+type Process struct {
+	PPid  uint32    `json:"ppid"`
+	Pid   uint32    `json:"pid"`
+	Tid   uint32    `json:"tid"`
+	Comm  [TASK_COMM_LEN]byte `json:"comm"`
+	PComm [TASK_COMM_LEN]byte `json:"pcomm"`
```

Comment:
Can't it be `string`?

Reply:
I didn't notice any difference, as I couldn't retrieve any values from them.
updated it with `string` just in case.